internal/controller/policyexceptiondraft_controller.go

```diff
@@ -51,12 +51,19 @@ func (r *PolicyExceptionDraftReconciler) Reconcile(ctx context.Context, req ctrl
 	_ = log.FromContext(ctx)
 	_ = r.Log.WithValues("policyexceptiondraft", req.NamespacedName)
 
-	var exceptionDraft giantswarmExceptions.PolicyExceptionDraft
+	var exceptionDraft giantswarmPolicy.PolicyExceptionDraft
 	// This should be a flag
-	background := false
+	background := true
```

Comment:
Can you tell me more about the background decision here? I'm torn on it, what are your thoughts?

Reply:
I would like to leave it false so we have proper results if a Policy Exception is present.

Reply:
That would be my inclination too. Why is it true right now?